src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -30,8 +31,22 @@ This is recorded in this file as an inner product space instance on `pi_Lp 2`.
 - `linear_isometry_equiv.of_inner_product_space`: provides an arbitrary isometry to Euclidean space
   from a given finite-dimensional inner product space, induced by choosing an arbitrary basis.
 
+- `std_orthonormal_basis`: the canonical `orthonormal_basis` on `euclidean_space 𝕜 ι`
```

Comment:
```suggestion
- `std_orthonormal_basis`: provides an arbitrarily-chosen orthonormal basis of a given
  finite-dimensional inner product space
```
Also need to delete the entries above and below on this list; they are subsumed now by the `orthonormal_basis` construction!

Reply:
Both lemmas added, and `std_orthonormal_basis` re-described.